src/OptionContainer.hpp

```diff
@@ -43,17 +43,24 @@ class OptionContainer
     std::string debuglevel;
     std::string path_debuglevel;
     DebugManager * myDebug;
-#endif 
+#endif
+
+    struct auth_entry {
+        int entry_id = 0;
```

Comment:
Codacy found an issue: [struct member 'auth_entry::entry_id' is never used.](https://app.codacy.com/manual/numsys/e2guardian/pullRequest?prid=6238856)